Doc/Tutorial/chapter_align.tex

```diff
@@ -1636,6 +1636,15 @@ \subsubsection{Basic usage}
 (see Section~\ref{sec:pairwise-aligner})
 stores the alignment parameters to be used for the pairwise alignments.
 
+These attributes can be set in the constructor of the object or after the object
+is made.
+
+%cont-doctest
+\begin{minted}{pycon}
+>>> aligner = Align.PairwiseAligner(match_score=2.0)
+>>> aligner.match_score = 1.0
```

Comment:
Perhaps for clarity it's better to set the same attribute to the same value, or alternatively to set a different attribute. I.e. either
```
These attributes can be set in the constructor of the object or after the object
is made.

%cont-doctest
\begin{minted}{pycon}
>>> aligner = Align.PairwiseAligner(match_score=2.0)
>>> aligner.mismatch_score = 1.0
\end{minted}
```
or


```
These attributes can be set in the constructor of the object or after the object
is made.

%cont-doctest
\begin{minted}{pycon}
>>> aligner = Align.PairwiseAligner(match_score=2.0)
\end{minted}{pycon}

or, equivalently,

%cont-doctest
\begin{minted}{pycon}
>>> aligner = Align.PairwiseAligner()
>>> aligner.match_score = 2.0
\end{minted}
```

Reply:
Just use the same value in the examples then?

Reply:
Yes, I will, but let's continue this PR after #2570 and #2548 have been merged.